lib/restrictions.js

```diff
@@ -73,6 +73,25 @@ export async function insert({restriction} = {}) {
   return record;
 }
 
+/**
+ * Inserts multiple restrictions into database.
+ *
+ * @param {object} options - The options to use.
+ * @param {Array} options.restrictions - An array of restrictions.
+ * @returns {Promise} - Settles once the operation completes.
+ */
+export async function bulkInsert({restrictions} = []) {
+  const operations = [];
+  for(const restriction of restrictions) {
+    const now = Date.now();
+    operations.push({
+      insertOne: {document: {restriction, meta: {created: now, updated: now}}}
```

Comment:
I'm assuming using bulkWrite here was intended, but if the only operation we are doing in bulk is `insertOne` then why not just use `insertMany`? Are we leaving ourselves open to doing multiple different operation types like `updateOne`, `deleteOne`, etc in a BulkWrite in the future?